angular/PdfViewer/Text-Markup-Annotation.md

```diff
@@ -50,21 +50,21 @@ export class DefaultComponent {
 
 The colorpicker control is provided in the text markup annotation tools to select the desired color for the text markup annotation to be added in the document.
 
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
+![Adding notes in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
 
 **Adding notes to the text markup annotation**
 
 The notes can be added to the text markup annotations using the context menu provided in the PDF viewer component.
 
 The below screenshot shows that the notes have been added to a text markup annotation in the PDF document.
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img3.png)
+![Editing the text markup in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img3.png)
```

Comment:
Adding notes to text markup annotation in Angular PDF Viewer

Reply:
Updated